infra/modules/service/variables.tf

```
@@ -97,3 +97,9 @@ variable "min_capacity" {
   type        = number
   default     = 2
 }
+
+variable "api_auth_token" {
```

⭐⭐  &nbsp; _(optional but seriously consider)_

```suggestion
variable "api_auth_token" {
  sensitive   = true
```